.github/workflows/bench.yml

```diff
@@ -86,6 +90,8 @@ jobs:
         path: ~/.cabal/cabal.tar.gz
 
   bench-example:
+    needs: pre_job
+    if: ${{ needs.pre_job.outputs.should_skip != 'true' }}
     needs: [bench-init]
```

Comment:
Is it possible to have two `needs` keys? I thought yaml is just more elaborated json...

Reply:
In the case that `bench-init` is skipped, is `bench-example` still reachable if we don't add this?

Reply:
It will skip its computation but from the GitHub's point of view I think the jot itself is not skipped, and will execute following jobs...

Reply:
Even if the action lets you duplicate keys it is better to have only one, will change